lib/robots/dor_repo/gis_assembly/generate_geo_metadata.rb

```diff
@@ -26,45 +26,45 @@ def perform(druid)
             return
           end
 
-          fn = Dir.glob("#{rootdir}/temp/**/*-iso19139.xml").first
```

Comment:
not only is `fn` so short that it has no meaning, but it was reused for multiple meanings.